stablehlo/reference/ProcessGrid.h

```diff
@@ -66,6 +68,9 @@ class RendezvousResult {
   /// `processId`. If key is not found, return an empty `Tensor`.
   Tensor lookup(ProcessId processId);
 
+  /// Returns the tensor values of the map `result_`.
```

Comment:
Also, this comment needs more details, specifically around the order in which tensor values are returned.